### ISSUE TITLE
Error on compilation

### ISSUE THREAD

**Issue by rodiram** [iminuit/iminuit#259]
Hello, I have this error when I install iminuit, I am using python 2.7, should I switch to python 3 ?
Thank you 
```
python setup.py install 

Compiling iminuit/_libiminuit.pyx because it changed.
Cythonizing iminuit/_libiminuit.pyx

Error compiling Cython file:
------------------------------------------------------------
...
                     'but error matrix is not accurate.')
            start = self.values[vname]
            sigma = self.errors[vname]
            bound = (start - bound * sigma, start + bound * sigma)

        values = np.linspace(*bound, bins, dtype=np.double)
                                        ^
------------------------------------------------------------

iminuit/_libiminuit.pyx:1066:41: Non-keyword arg following star-arg
Traceback (most recent call last):
  File "setup.py", line 103, in <module>
    extensions = cythonize(extensions)
  File "/home/rodiram/prog/anaconda/lib/python2.7/site-packages/Cython/Build/Dependencies.py", line 825, in cythonize
    cythonize_one(*args[1:])
  File "/home/rodiram/prog/anaconda/lib/python2.7/site-packages/Cython/Build/Dependencies.py", line 944, in cythonize_one
    raise CompileError(None, pyx_file)
Cython.Compiler.Errors.CompileError: iminuit/_libiminuit.pyx
```

**Comment by cdeil** [comment]
I think it's this recently written line, which indeed is only valid on Python 3, not Python 2:
https://github.com/iminuit/iminuit/blame/ab0f420e4fdc4bb0f668a2ee45aa5d331f4d73a7/iminuit/_libiminuit.pyx#L1069

@HDembinski - Shouldn't the same error that @rodiram got have also been triggered on the Python 2.7 CI build when running Cython? Maybe rewrite that line to make it work on Python 2 while we still support that?

**Comment by cdeil** [comment]
This is a bit of a weird issue.

Indeed, we had code in `iminuit/_libiminuit.pyx` that's Python 3 only syntax.
Should be fixed in #262.

Now why did this show up for you, but not in our Python 2.7 CI build here?
https://travis-ci.org/iminuit/iminuit/jobs/400301678#L549

My guess it that you have an older version of Cython, and recent version of Cython just accept this PYthon 3 syntax and compile it to Python 2 code when running on Python 2 somehow.

I wasn't able to reproduce the issue locally (didn't want to spend time installing older Cython versions), so I'm not sure #262 is a complete fix. But I think it's harmless, just you as a developer have to either update to Python 3 or a more recent Cython version.

**Comment by rodiram** [comment]
Thank you for this quick answer Christoph !!
Indeed I was using an old Cython version 0.21, iMinuit installed nicely with version 0.28.3
